bom/pom.xml

```diff
@@ -882,11 +882,6 @@
                 <artifactId>helidon-integrations-cdi-oci-objectstorage</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
```

Comment:
Changes in this file look wrong.

Reply:
Sorry, which change? We are making atp module available to both se/mp bundle, just like objectstorage/vault/helath etc.

Reply:
Also, deps were arranged alphabetically.